src/main/java/internet/store/dao/GenericDao.java

```
@@ -1,10 +1,11 @@
 package internet.store.dao;
 
+import java.sql.SQLException;
 import java.util.List;
 import java.util.Optional;
 
 public interface GenericDao<T, K> {
-    T create(T item);
+    T create(T item) throws SQLException;
```

should we throw an exception here?